blog/2022-1-2-your-first-extension.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: your-first-extension
+title: Your first extension
+authors: [CharlieS1103]
```

Comment:
First of all, you need to pass the build process, which you can do by adding yourself as a valid author in https://github.com/afonsojramos/spicetify-docs/blob/main/blog/authors.yml. Please use a lowercase username.